SFS_binaural_synthesis/ir_generic.m

```diff
@@ -85,7 +85,9 @@
     % If needed interpolate the given impulse response set and weight, delay the
     % impulse for the correct distance
     ir = get_ir(sofa,X,[phi 0],x0(ii,1:3),'cartesian',conf);
-
+    % Correct Green's function amplitude
+    ir = ir/(4*pi);
```

Comment:
I think the HRTFs should have the amplitude when listening to a point source with an amplitude of 1/(4*pi) that is representing the 3D free-field Green's function when I simulate SFS. Maybe this factor should be applied somewhere else if this function is used for other purposes as well?